docs/source/getting_started/installation.md

```diff
@@ -3,7 +3,7 @@
 ### Installing from PyPI
 
 If you already have [PyTorch](https://pytorch.org/) or [Tensorflow](https://www.tensorflow.org) installed on your machine, 
-Quantus can be obtained from [PyPI](https://pypi.org/project/quantus/) as follows:
+the most light-weight version of Quantus can be obtained from [PyPI](https://pypi.org/project/quantus/) as follows:
```

Comment:
maybe add something like
"This means that additional explainability functionality will not be included."

Reply:
added.